### DISCUSSION TITLE
Is this working for anyone?

### DISCUSSION THREAD

**Discussion by stayupthetree** [seanap/beets-audible#14] [Q&A]
I got it installed and beets is searching, but when I do get the right artist and book, its the spanish version OR I get absolutely rubbish results, like barely a 7% match

Got a higher % here, but definitely the wrong book
![image](https://github.com/user-attachments/assets/90e6e4f8-4620-4c2d-8ba4-b0f1bccdf982)

**Comment by AP0LL0-19** [comment]
I have the same problem, the books it matches are way off. Did u manage to find a fix ?
